theano/tensor/opt.py

```diff
@@ -5186,7 +5190,7 @@ def local_sum_prod_mul_by_scalar(node):
                 ret = T.mul(*mul_inputs)
                 # Copy over stacktrace from previous output to new mul op,
                 # for same reason as above.
-                copy_stack_trace(node.outputs, [ret] + mul_inputs)
+                copy_stack_trace(node.outputs, ret+mul_inputs)
```

Comment:
Same here, `ret` is a variable, and `mul_inputs` is a list.
This actually makes some tests fail.